src/views/states/list/components/InterfacesTable.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC } from 'react';
+
+import { Table, TableGridBreakpoint, TableHeader, Tbody } from '@patternfly/react-table';
+import { NodeNetworkConfigurationInterface } from '@types';
+
+import useInterfaceColumns from '../hooks/useInterfaceColumns';
+
+import InterfacesTypeSection from './InterfacesTypeSection';
+
+import './interfaces-table.scss';
+
+interface InterfacesTableProps {
+  interfacesByType: { [interfaceType in string]: NodeNetworkConfigurationInterface[] };
+}
+
+const InterfacesTable: FC<InterfacesTableProps> = ({ interfacesByType }) => {
+  const [_, activeColumns] = useInterfaceColumns();
+
+  return (
+    <Table
+      cells={activeColumns}
+      gridBreakPoint={TableGridBreakpoint.none}
+      role="presentation"
+      className="interfaces-table"
+    >
+      <TableHeader />
+      <Tbody>
```

Comment:
Shouldn't be here `TableBody` instead? According to the doc https://www.patternfly.org/2020.04/documentation/react/components/table